### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by metricrb** [noblox/noblox.js#382] [Q&A] [answered]
I'm not the best coder but how would I make it so once a person states someones username it converts to ID and ranks them to a certain rank?

I tried this and got error 400.

```  
 let id = noblox.getIdFromUsername(user)   
        noblox.setRank(6621899, id, "Citizen") 
```

**Comment by suufi** [comment, marked as answer]
Please read the page on the [documentation](https://noblox.js.org/tutorial-Promises.html) about promises.
